src/pudl/metadata/fields.py

```diff
@@ -736,6 +784,10 @@
         "type": "string",
         "description": "The docket number relating to the FERC qualifying facility exempt wholesale generator status.",
     },
+    "ferc_license_id": {
+        "type": "string",
+        "description": "The FERC license ID of a project.",
+    },
```

Comment:
I think there is already a field called `license_id_ferc1`. I'm guessing they are referring to the same thing?